packages/component-boilerplate/src/environment/BatchedGraphQLClient.ts

```diff
@@ -0,0 +1,286 @@
+import { Subject } from "rxjs";
+import { bufferTime } from "rxjs/operators";
+import { v4 as uuid } from "uuid";
```

Comment:
@sam-beveridge should use `nanoid` instead of `uuid`

Reply:
👍